src/components/final-form/RadioFields/ComplexRadioFields.tsx

```diff
@@ -26,10 +29,14 @@ function ComplexRadioFields<TYPE>({
   options,
   optionLabelField,
   validate,
+  isRequired,
   ...restProps
 }:PropsWithChildren<Props<TYPE>>) {
-  const { input: { onChange, value }, meta } = useField(name, {
-    validate
+  const { input: { onChange, value }, meta } = useField<TYPE>(name, {
```

Comment:
Is the generic type needed? The `<TYPE>` part.

Reply:
Not really, I removed it